tests/test_claude_logs.py

```
@@ -0,0 +1,307 @@
+    def test_session_event_creation(self):
+        """Test creating a SessionEvent object."""
+        timestamp = datetime.now()
+        event = SessionEvent(
+            timestamp=timestamp,
+            session_id="test_session",
+            directory="/test/dir",
+            message_type="user",
+            content_preview="Test message",
+            uuid="test-uuid-123",
+        )
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)